packages/react/src/components/Header/_header.scss

```diff
@@ -31,6 +58,56 @@
     border-color: $ui-01;
     outline: none;
   }
+
+  .#{$iot-prefix}--header__overflow-menu-container {
+    position: relative;
+  }
+
+  .#{$prefix}--overflow-menu {
+    &:hover {
+      background-color: $gray-90;
```

Comment:
Not sure we need to support this, but is there a non color specific carbon name for the gray-90 that works with the different themes? E.g. `$hover-secondary`. Perhaps this is a question for the design team. It would be nice if the invision mocks came with the name of the colors.

Reply:
Nope, not that I can find. grepping through all the stuff just shows carbon using the raw variables in ui-shell, too. 
```
./node_modules/@ai-apps/styles/scss/vendor/carbon-components/scss/components/ui-shell/_navigation-menu.scss:    background-color: $ibm-color__gray-90;
./node_modules/@ai-apps/styles/scss/vendor/carbon-components/scss/components/ui-shell/_theme.scss:/// TODO needs color var gray-90-hover
./node_modules/@ai-apps/styles/scss/vendor/carbon-components/scss/components/ui-shell/_theme.scss:$shell-header-bg-06: $carbon--gray-90;
./node_modules/@ai-apps/styles/scss/vendor/carbon-components/scss/components/ui-shell/_theme.scss:$shell-panel-bg-04: $carbon--gray-90;
./node_modules/@ai-apps/styles/scss/vendor/carbon-components/scss/components/ui-shell/_theme.scss:/// TODO needs color var gray-90-hover
```